### PR TITLE
FOGL-7495: Fledge:Audit Log "NTFSN" generated even when delivery fails

### PR DESCRIPTION
**Pull request by Innovative-ashwin**

### CODE REVIEW

File: customasset.cpp [hunk 1]
```diff
@@ -81,13 +81,13 @@ CustomAsset::~CustomAsset()
  * @param triggerReason		Why the notification is being sent
  * @param message		The message to send
  */
```
Comment (gnandan) [inline customasset.cpp:83]: No return type ??

File: customasset.cpp [hunk 2]
```diff
@@ -81,13 +81,13 @@ CustomAsset::~CustomAsset()
...
-void CustomAsset::notify(const string& notificationName, const string& triggerReason, const string& message)
+bool CustomAsset::notify(const string& notificationName, const string& triggerReason, const string& message)
 {
 	vector<Datapoint *>	datapoints;
 
 	if (!m_ingest)
 	{
-		return;
+		return false;
```
Comment (gnandan) [inline customasset.cpp:90]: Can we log some message here why retuning false ?

File: customasset.cpp (hunk 1 above)
Comment (Innovative-ashwin) [reply customasset.cpp:83]: fixed

File: customasset.cpp (hunk 2 above)
Comment (Innovative-ashwin) [reply customasset.cpp:92]: fixed

**Pull request merged**